admin/app/components/solidus_admin/ui/forms/address/component.rb

```diff
@@ -1,13 +1,15 @@
 # frozen_string_literal: true
 
 class SolidusAdmin::UI::Forms::Address::Component < SolidusAdmin::BaseComponent
-  def initialize(form:, disabled: false)
+  def initialize(form:, address:, disabled: false, html_attributes: {})
```

Comment:
If we're using a string name in the `form:` I suggest renaming the kwarg to `name:` so it will be consistent and similar to using any input field. 

Using `name_prefix:` would also be an option, but I like `name:` more: it's shorter will act as the (base) name for the whole sub-form.